src/app/app.component.ts

```diff
@@ -51,6 +54,17 @@ export class AppComponent implements OnInit {
                 private userService: UserService,
                 private snack: MatSnackBar) {
 
+
+        //check for width
+        if (window.screen.width >= 768) {
```

Comment:
You should use ObservableMedia instead, see https://github.com/Supamiu/ffxiv-teamcraft/blob/refactor/src/app/modules/item/item/item.component.ts#L285. Documentation for ObservableMedia: https://github.com/angular/flex-layout/wiki/Responsive-API